### ISSUE TITLE
Tagged Releases

### ISSUE THREAD

**Issue by mgrecar** [sekai-soft/rss-lambda#3]
Hi,

Would you be able to issue releases when there's a new image published?  Just using `latest` and only keeping one version around makes it hard to know if I've updated my container, and very difficult to automatically monitor for updates.

**Comment by KTachibanaM** [comment]
Images with git commit sha tags should be pushed now https://github.com/sekai-soft/rss-lambda/pkgs/container/rss-lambda

**Comment by mgrecar** [comment]
Yeah, that's not really what I was getting at.  You can already pin a container version with the `@sha256` notation, like `image: ghcr.io/sekai-soft/rss-lambda@sha256:e9beed8700153c0db2461d6d09bb342123f2fa60d7b167695c7840b98f7a1883`.  Adding a tag with that same data isn't really too helpful.  

This was more about adding release versioned tags so you can more easily see there was a new release.  Right now, commits just get pushed to main and it builds a new version, but it's not obvious that's updated.  I was thinking issuing a GitHub release with changes, or at least adding a human sensible version label to identify changes would be more useful than understanding commit `e9beed8700153c0db2461d6d09bb342123f2fa60d7b167695c7840b98f7a1883` is an update from `b447fc6bba994eb46884c07ebeed5eab7a2d934f9e01881d609f2663970cefa9`.

No big deal for me though, I've worked around monitoring for updates with other tools.
